index.js

```diff
@@ -22,8 +22,11 @@ export default {
             show: 'نمایش',
             sort: 'مرتب‌سازی',
             undo: 'لغو',
+            unselect: 'عدم انتخاب',
             expand: 'باز کن',
-            close: 'ببند',
```

Comment:
close & expand translations don't match as opposites. Please fix it.

Reply:
i have Reverted "change ra.action.close translation" commit